### DISCUSSION TITLE
如何规避一到0点就重置的问题

### DISCUSSION THREAD

**Discussion by piderowiki** [simple-robot/Discussions#11] [Q&A]
我把机器人部署到了一个我平时不怎么用的电脑上，但是第二天起来发现服务停了。上电脑一看发现，在23h55min的时候服务器重置了一波，一直到我重新打开idea机器人才尝试了重连，，，请问这样的有解决方法吗?

**Comment by piderowiki** [comment]
还是说这是因为我网络的问题吗?

**Reply by ForteScarlet** [reply]
一般来讲并没有什么0点重置的说法，试着检查网络环境或其他可能的因素
